routers/admin/users.go

```diff
@@ -77,11 +77,12 @@ func NewUserPost(ctx *context.Context, form auth.AdminCreateUserForm) {
 	}
 
 	u := &models.User{
-		Name:      form.UserName,
-		Email:     form.Email,
-		Passwd:    form.Password,
-		IsActive:  true,
-		LoginType: models.LoginPlain,
+		Name:               form.UserName,
+		Email:              form.Email,
+		Passwd:             form.Password,
+		IsActive:           true,
+		LoginType:          models.LoginPlain,
+		MustChangePassword: true,
```

Comment:
Maybe a form checkbox default checked on UI is better?

Reply:
@lunny there is other PR that adds that

Reply:
OK

Reply:
@lunny see https://github.com/go-gitea/gitea/pull/4563